dependencies/pre-deployment-pvc-binding/pre-deployment-pvc-binding.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-pvc-ns
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-pvc
+  namespace: test-pvc-ns
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: v1
```

Comment:
I think it's sufficient to just create the pvc and ensure it's bound. The pod is not needed.
I also don't see that the script is doing anything with the pod.

Reply:
I am not sure it will be bound unless it will have a pod that uses it

Reply:
The default storage class in kind has volume binding mode of wait to first customer. It won't bind unless the pod is there. It binds before the pod starts, so the script doesn't have to wait for the image to be downloaded etc., but it needs the pod to be created.